anitya/lib/backends/maven.py

```diff
@@ -70,3 +75,37 @@ def get_versions(cls, project):
                           .replace(':', '/'))
 
         return get_versions_by_regex(url, REGEX, project)
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 40 packages to Maven central index
+
+        by calling maven-index-checker application
+        '''
+        maven_url = 'http://repo2.maven.org/maven2'
+        jar_path = '/src/maven-release-checker-1.0-SNAPSHOT-jar-with-dependencies.jar'
+        # session for checking whether there is already created package
+        session = init(APP.config['DB_URL'], create=True, debug=False)
```

Comment:
``create=True`` will cause SQLAlchemy to emit create statements which I don't think we really want to do here.

FYI when https://github.com/release-monitoring/anitya/pull/466/commits/d8c789b3f512c01560518f6a89798885a86fb202 is merged you can just
```
from anitya.lib.model import Session
session = Session()
session.query.....
Session.remove()
```

In fact, if all you're doing is querying the projects, you'll be fine with
```
from anitya.lib.model import Project
try:
    projects = Project.query.filter(Project.version_url == version_url).all()
except NoResultFound:
    projects = None
```